src/schoology-extractor/tests/data_generation/main.py

```diff
@@ -0,0 +1,28 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Is the plan to move this to `/utils/something`? Alternately could leave it in the extractor as `test_data_generatory.py` sitting next to `main.py`, if we think that downstream users might be interested and if that simplifies package / module references.

Reply:
My thinking behind having it under `tests` is that it's essentially a generator of test data.  Not the best fit, but a reasonable first cut at a location.